numpy/ma/core.py

```diff
@@ -4107,10 +4107,7 @@ def __repr__(self):
                 'dtype': str(self.dtype)
             }
             is_structured = bool(self.dtype.names)
-            key = '{}_{}'.format(
-                'long' if is_long else 'short',
-                'flx' if is_structured else 'std'
-            )
+            key = f'{"long" if is_long else "short"}_{"flx" if is_structured else "std"}'
```

Comment:
```suggestion
            key = "_".join(["long" if is_long else "short", "flx" if is_structured else "std"])
```

The original `.format()` version is arguably the best of the three alternatives, IMO.